src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6448,6 +6444,7 @@ void UpdateSysSizing(EnergyPlusData &state, Constant::CallIndicator const CallIn
                 CoolTimeStepNum = state.dataSize->TermUnitFinalZoneSizing(TermUnitSizingIndex).TimeStepNumAtCoolMax;
                 OutAirTemp += state.dataSize->DesDayWeath(CoolDDNum).Temp(CoolTimeStepNum) * coolMassFlow / (1.0 + termUnitSizing.InducRat);
                 OutAirHumRat += state.dataSize->DesDayWeath(CoolDDNum).HumRat(CoolTimeStepNum) * coolMassFlow / (1.0 + termUnitSizing.InducRat);
+                saveCoolTUSizingIndex = TermUnitSizingIndex;
```

Comment:
Will save the index of the last TU scanned for this air loop. Since the zone peak will likely occur at different times, this will report one of the zones (the last one scanned) peak time.